### ISSUE TITLE
BatchNormalization and SpatialBatchNormalization conversion missing

### ISSUE THREAD

**Issue by jorditg** [soumith/cudnn.torch#131]
Doesn't convert nn.BatchNormalization to cudnn.BatchNormalization. The same for Spatial.

**Comment by szagoruyko** [comment]
don't have the conversion because the format of running_var is different to avoid bugs. will add the conversion when cudnn is fixed to behave as nn.

**Comment by oeway** [comment]
+1

**Comment by davek44** [comment]
+1

**Comment by szagoruyko** [comment]
it is supported in R5 bindings.

**Comment by szagoruyko** [comment]
closing because R5 is master now.

**Comment by gokceneraslan** [comment]
Could you please also add BatchNorm support to Readme.md?

**Comment by soumith** [comment]
@gokceneraslan added!
